build_tools/ci/run_matmul_test.sh

```diff
@@ -28,16 +28,16 @@ if [ "$#" -lt 2 ] || [ "$#" -gt 5 ]; then
    # The expected parameters are
    #    1) <output-dir>            (required)
    #    2) <iree-install-dir>      (required)
-   #    4) <peano-install-dir>     (optional)
+   #    3) <peano-install-dir>     (optional)
+   #    4) <vitis-install-dir>     (optional)
    #    5) <xrt-dir>               (optional)
-   #    6) <vitis-install-dir>     (optional)
     echo -e "Illegal number of parameters: $#, expected 2-5 parameters." \
             "\n The parameters are as follows:" \
             "\n     1) <output-dir>               (required)" \
             "\n     2) <iree-install-dir>         (required)" \
             "\n     3) <peano-install-dir>        (optional)" \
-            "\n     4) <xrt-dir>                  (optional)" \
-            "\n     5) <vitis-install-dir>        (optional)" \
+            "\n     4) <vitis-install-dir>        (optional)" \
+            "\n     5) <xrt-dir>                  (optional)" \
```

Comment:
Why do you change the order of 4 and 5? Just want to make sure the parameters are passing in the same order between `run_matmul_test.sh` and `run.py`.

Reply:
Because after this PR XRT becomes optional (i.e., unused on Linux) but Vitis remains so placing `xrt-dir` fifth means it can be omitted. For `run.py`, the directory is already optionally passed using `--xrt-dir`.